test/unit_tests/documentFormatting/documentFormatting.test.ts

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { beforeEach, expect, jest, test } from '@jest/globals';
```

Comment:
Are all these disabled really necessary?

Reply:
Do you really want to know?